google-cloud-clients/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java

```diff
@@ -197,7 +197,7 @@ public String getTopicNameString() {
     }
 
     message = messageTransform.apply(message);
-    OutstandingBatch batchToSend = null;
+    List<OutstandingBatch> batchesToSend = new ArrayList<>();
```

Comment:
IIRC, this used to be a LinkedList. Is changing to an ArrayList ok?

Reply:
This was never a `LinkedList`, AFAIK.  Historically, there was only `OutstandingBatch batchToSend = null`

Reply:
sorry - was confusing it was MessagesBatch.